crates/sui-core/src/authority_aggregator.rs

```diff
@@ -1071,6 +1083,15 @@ where
         }
     }
 
+    fn record_rpc_error_maybe(&self, name: ConciseAuthorityPublicKeyBytesRef, error: &SuiError) {
+        if let SuiError::RpcError(message, _code) = error {
+            self.metrics
+                .total_rpc_err
+                .with_label_values(&[&name.to_string(), message.as_str()])
```

Comment:
I happened to have a look on this metric in testnet during some investigation and observed that the `error_message` field has a very [high cardinality](https://metrics.sui.io/explore?orgId=1&left=%7B%22datasource%22:%228Xt1pVoVk%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22datasource%22:%7B%22type%22:%22prometheus%22,%22uid%22:%228Xt1pVoVk%22%7D,%22editorMode%22:%22code%22,%22expr%22:%22count%20by%20%28error_message%29%20%28total_rpc_err%7Bnetwork%3D%5C%22testnet%5C%22%7D%29%22,%22legendFormat%22:%22__auto%22,%22range%22:true,%22instant%22:true%7D%5D,%22range%22:%7B%22from%22:%22now-1h%22,%22to%22:%22now%22%7D%7D) on its values (currently 7K+ ) due to messages like:
```
Object (0xf0cd6892f77e57705fc71a24faffbb0a1b49d4ed76440139c1f5e02681164eff, SequenceNumber(7594531), o#9L7XE8JUrfVA49JkjGXCBrbPdpnhKuYJjNcJACkwadYa) already locked by a different transaction: TransactionDigest(ETRttQoeN9WpoD9xtmgvfThNzuiEc9jcB8PvQtLtSovn)
```

which is expected, as error message isn't static. I believe this will eventually create a saturation in our metrics infra. @longbowlu we could probably do some "anonymisation" on the variable parts so cardinality could be reduced - or emit the info as `err` message instead.